docs/onboarding_admins.rst

```diff
@@ -0,0 +1,97 @@
+Onboard Additional Admins
+=========================
+
+If you are the only admin for your SecureDrop, you can skip this chapter.
+It instructs you how to create additional *Admin Workstation* USB drives.
+Each *Admin Workstation* will have its own SSH keypair.
+
+This chapter assumes that you have one working *Admin Workstation*. If you've
+not completed that part of the setup yet, see :doc:`set_up_admin_tails`.  If
+your *Admin Workstation* is corrupted or lost, and you don't have a
+:doc:`backup <backup_workstations>`, see :doc:`rebuild_admin`.
+
+.. important::
+
+   If you make configuration changes on your servers using one
+   *Admin Workstation*, they may be overwritten by another *Admin Workstation*
+   if its local copy of the configuration is not identical. When working
+   with multiple admins, it is therefore important to establish protocols
+   for coordinating configuration changes. See :ref:`multiple_admins`.
+
+To onboard an additional administrator, you will need:
+
+- your existing *Admin Workstation* USB drive (referred to as **AW1** below)
+- an additional empty USB drive (referred to as **AW2** below)
+
+To set up AW2, follow these steps:
+
+1. Boot AW1, `unlock its persistent volume <https://tails.boum.org/doc/first_steps/persistence/use/index.en.html>`__,
+   and `set an admin password on the welcome screen <https://tails.boum.org/doc/first_steps/welcome_screen/administration_password/>`__
+2. Ensure that Tails and the SecureDrop version on AW1 are up-to-date.
+   If not, update now.
```

Comment:
Because we have so much documentation (:sweat:), it might be friendly to link to the latest upgrade guide here to eliminate confusion (or, if keeping the 'latest' upgrade guide linked is a hassle, at least to mention "update now by following the `|version|` upgrade guide in the "Upgrade SecureDrop" section of these docs."

Reply:
Agreed. What I've done for now is to just add a symbolic reference in the latest upgrade guide. We will have to move that whenever we initialize a new upgrade guide, but the linter will yet at us if we try to merge docs changes with either two duplicate symbols or none, so it should be fairly self-evident to anyone adding the migration guide what needs to be done.